### PR TITLE
Updating admin password on Clusters with CRDB

### PR DESCRIPTION
**Pull request by AlonMagrafta**
not bounded to version, can be merged to latest once approved

### CODE REVIEW

File: content/rs/security/admin-console-security/user-security.md [hunk 1]
```diff
@@ -187,6 +187,14 @@ rladmin cluster config cm_session_timeout_minutes <number_of_min>
 
 Here, number_of_min is the number of minutes after which sessions will time out.
 
+## Updating Administrator password on Clusters with Active-Active databases
+
+Active-Active databases are relying on the Administrator credentials for management operations of the Active-Active database.
```
Comment (kaitlynmichael) [inline content/rs/security/admin-console-security/user-security.md:192]:
```suggestion
Active-Active databases are relying on the administrator credentials for management operations of the Active-Active database.
```

File: content/rs/security/admin-console-security/user-security.md [hunk 2]
```diff
@@ -187,6 +187,14 @@ rladmin cluster config cm_session_timeout_minutes <number_of_min>
...
+When updating an administrator user password on a cluster with Active-Active databases please follow these steps:
+1. From the user management page, update the administrator user password on the cluster/s you wish to update.
+1. Go to each participating cluster and for each Active-Active database update the Admin user/s credentials that have you changed on step 1.
+Do not perform any management operation on the databases between these two steps.
```
Comment (kaitlynmichael) [inline content/rs/security/admin-console-security/user-security.md:196]:
```suggestion
{{< warnging> }}Do not perform any management operation on the databases between these two steps. {{< /warning >}}
```

File: content/rs/security/admin-console-security/user-security.md [hunk 3]
```diff
@@ -187,6 +187,14 @@ rladmin cluster config cm_session_timeout_minutes <number_of_min>
 
 Here, number_of_min is the number of minutes after which sessions will time out.
 
+## Updating Administrator password on Clusters with Active-Active databases
+
+Active-Active databases are relying on the Administrator credentials for management operations of the Active-Active database.
+When updating an administrator user password on a cluster with Active-Active databases please follow these steps:
+1. From the user management page, update the administrator user password on the cluster/s you wish to update.
+1. Go to each participating cluster and for each Active-Active database update the Admin user/s credentials that have you changed on step 1.
```
Comment (kaitlynmichael) [inline content/rs/security/admin-console-security/user-security.md:195]:
```suggestion
1. Go to each participating cluster and for each Active-Active database update the admin user credentials that have you changed with the last step. 
```

**Review by kaitlynmichael — CHANGES_REQUESTED** [review]

File: content/rs/security/admin-console-security/user-security.md [hunk 4]
```diff
@@ -187,9 +187,16 @@ rladmin cluster config cm_session_timeout_minutes <number_of_min>
 
 Here, number_of_min is the number of minutes after which sessions will time out.
 
+## Updating Administrator password on Clusters with Active-Active databases
+
+Active-Active databases are relying on the administrator credentials for management operations of the Active-Active database.
+When updating an administrator user password on a cluster with Active-Active databases please follow these steps:
+1. From the user management page, update the administrator user password on the cluster/s you wish to update.
```
Comment (lanceleonard) [inline content/rs/security/admin-console-security/user-security.md:194]:
```suggestion
1. From the user management page, update the administrator user password on the clusters you wish to update.
```

File: content/rs/security/admin-console-security/user-security.md [hunk 5]
```diff
@@ -187,9 +187,16 @@ rladmin cluster config cm_session_timeout_minutes <number_of_min>
 
 Here, number_of_min is the number of minutes after which sessions will time out.
 
+## Updating Administrator password on Clusters with Active-Active databases
+
+Active-Active databases are relying on the administrator credentials for management operations of the Active-Active database.
```
Comment (lanceleonard) [inline content/rs/security/admin-console-security/user-security.md:192]:
```suggestion
Active-Active databases use administrator credentials to manage operations for Active-Active database.
```

File: content/rs/security/admin-console-security/user-security.md [hunk 6]
```diff
@@ -187,9 +187,16 @@ rladmin cluster config cm_session_timeout_minutes <number_of_min>
...
+When updating an administrator user password on a cluster with Active-Active databases please follow these steps:
```
Comment (lanceleonard) [inline content/rs/security/admin-console-security/user-security.md:193]:
```suggestion
To update the administrator user password on a cluster with Active-Active databases:
```

File: content/rs/security/admin-console-security/user-security.md [hunk 7]
```diff
@@ -187,9 +187,16 @@ rladmin cluster config cm_session_timeout_minutes <number_of_min>
...
+1. From the user management page, update the administrator user password on the cluster/s you wish to update.
+1. Go to each participating cluster and for each Active-Active database update the admin user credentials that have you changed with the last step.
```
Comment (lanceleonard) [inline content/rs/security/admin-console-security/user-security.md:195]:
```suggestion
1. For each participating cluster _and_ each Active-Active database, update the admin user credentials to match the changes in Step 1. 
```

File: content/rs/security/admin-console-security/user-security.md [hunk 8]
```diff
@@ -187,9 +187,16 @@ rladmin cluster config cm_session_timeout_minutes <number_of_min>
 
 Here, number_of_min is the number of minutes after which sessions will time out.
 
+## Updating Administrator password on Clusters with Active-Active databases
+
+Active-Active databases are relying on the administrator credentials for management operations of the Active-Active database.
+When updating an administrator user password on a cluster with Active-Active databases please follow these steps:
+1. From the user management page, update the administrator user password on the cluster/s you wish to update.
+1. Go to each participating cluster and for each Active-Active database update the admin user credentials that have you changed with the last step. 
+{{< warnging> }}Do not perform any management operation on the databases between these two steps. {{< /warning >}}
```
Comment (lanceleonard) [inline content/rs/security/admin-console-security/user-security.md:196]:
```suggestion
{{< warning> }}Do not perform any management operation on the databases until these steps are complete. {{< /warning >}}
```

**Review by lanceleonard — APPROVED** [review]
LGTM (some suggested wording changes).

File: content/rs/security/admin-console-security/user-security.md [hunk 9]
```diff
@@ -187,9 +187,16 @@ rladmin cluster config cm_session_timeout_minutes <number_of_min>
 
 Here, number_of_min is the number of minutes after which sessions will time out.
 
+## Updating Administrator password on Clusters with Active-Active databases
```
Comment (lanceleonard) [inline content/rs/security/admin-console-security/user-security.md:190]:
```suggestion
## Update Active-Active cluster admin credentials
```

**Comment by AlonMagrafta** [comment]
LGTM

**Review by lanceleonard — APPROVED** [review]

**Pull request merged**